plugins/flashrom/fu-plugin-flashrom.c

```diff
@@ -264,6 +264,7 @@ fu_plugin_update (FuPlugin *plugin,
 	}
 
 	fu_device_set_status (device, FWUPD_STATUS_DEVICE_WRITE);
+	fu_device_set_progress (device, 0); /* urgh */
```

Comment:
we can't get anything out of the library?  Sounds like a development action for a future version.  Can we at least have time estimates that we can show?  Otherwise it looks hung.

Reply:
At the moment, no, but I've approached flashrom with a proposal, although nobody (including me) actually wrote any code yet: https://www.mail-archive.com/flashrom@flashrom.org/msg13519.html

Reply:
**Narrator**: _He wrote some code_